.github/workflows/create-offline-installer.yml

```
@@ -27,11 +27,19 @@ jobs:
           echo "staging_path=${staging_path}" >> $GITHUB_ENV
           export temp_path=/home/runner/work/spark/spark/var/tmp/spark
           echo "temp_path=${temp_path}" >> $GITHUB_ENV
-          export staging_archive=spark-offline-archive.run
-          echo "staging_archive=${staging_archive}" >> $GITHUB_ENV
           echo "registry=registry.scality.com" >> $GITHUB_ENV
           echo "img=fieldreportservice" >> $GITHUB_OUTPUT
           echo "image=${registry}/${image_path}" >> $GITHUB_OUTPUT
+      - name: Set the staging_archive name
+        run: |
+          export staging_archive=spark-offline-archive-${{ inputs.tag }}.run
+          echo "staging_archive=${staging_archive}" >> $GITHUB_ENV
+        if: ${{ ! inputs.prerelease }}
+      - name: Set the staging_archive name if prerelease
+        run: |
+          export staging_archive=spark-offline-archive-prerelease.run
```

Usually even when there's a prerelease, a tag is made, so it's better to include it.
```suggestion
          export staging_archive=spark-offline-archive-${{ inputs.tag }}-prerelease.run
```